src/vm/clsload.cpp

```diff
@@ -1489,7 +1489,7 @@ BOOL ClassLoader::FindClassModuleThrowing(
     {
         INSTANCE_CHECK;
         if (FORBIDGC_LOADER_USE_ENABLED()) NOTHROW; else THROWS;
-        if (FORBIDGC_LOADER_USE_ENABLED()) GC_NOTRIGGER; else GC_TRIGGERS;
```

Comment:
I think these contracts should stay as `if (FORBIDGC_LOADER_USE_ENABLED()) GC_NOTRIGGER; else GC_TRIGGERS;`.

Do you see it causing any problems?